runtime/moonbase/Cargo.toml

```diff
@@ -245,7 +245,9 @@ runtime-benchmarks = [
 	"xcm-builder/runtime-benchmarks",
 	"pallet-xcm/runtime-benchmarks",
 	"pallet-ethereum/runtime-benchmarks",
+	"pallet-asset-manager/runtime-benchmarks",
 ]
+moonbase-runtime-benchmarks = []
```

Comment:
Should this imply `runtime-benchmarks` so you don't need multiple feature flags to use it? You would probably need to do that in a lot of pallets or create an alias in `.cargo/config.toml`, so maybe not...

Reply:
I can try

Reply:
The main issue here is that I need to propagate the flag at least till runtime-common, which is the common place where we put our benchmark apis. And since runtime-common is common for all runtimes, I need to have a special flag to incorportate the benchmarks for pallets belonging to moonbase only.

The way I designed it currently is that if you build with moonbase-runtime-bechmarks, it will build with runtime-benchmarks & moonbase-runtime-benchmarks just the moonbase runtime.  From here there are two possibilities:

- Leave it as is, where building with features moonbase-runtime-bechmarks compile the moonbase runtime with both runtime-benchmarks and moonbase-runtime-benchmarks
- Make it so that we only have one flag in moonbase called moonbase-runtime-benchmarks, and do something like moonbase-runtime-benchmarks = ["frame-support/runtime-benchmarks", "pallet-society/runtime-benchmarks", etc..]"

Quite honestly I'd rather not be too invasive with this PR, and not change much how things worked. I think this approach is the lesser invasive as things worked as before, but we have a new flag with which we can run the just moonbase benchmarks


There are two options h